phpcs.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0"?>
```

Comment:
Could you elaborate why ?

This is the project standard to use and individual contributors should not overrule it with their own `phpcs.xml` file.